lib/tasks/forem.rake

```diff
@@ -3,8 +3,7 @@ namespace :forem do
   # Since we execute this tasks in bin/setup as well as app_initializer:setup
   # everything happening here needs to be idempotent.
   task setup: :environment do
-    # NOTE: There are currently no tasks here. This will change as soon as we
-    # add/update the task for seeding navigation links to support RFC #237.
+    Rake::Task["navigation_links:create"].execute
```

Comment:
I really have NO IDEA if this is the proper way to call the rake task in this file 🙈  Please advise!

Reply:
We tend to use `invoke` over `execute`. The difference between them is that `invoke` will also run the called task's dependencies. It shouldn't really make a difference in this case though.